ToDo/Program.cs

```diff
@@ -1,200 +1,191 @@
+        {
+            Task task = _listOfTasks.GetTaskList()[_positionInTask];
+            Console.Write($"\nEnter desription for {task.Title}: ");
+            string userInput = Console.ReadLine();
+            task.AddDescription(userInput);
+        }
+
+        public static void ExecuteStatusTask(string status)
+        {
+            if (status == "done")
+            {
+                _listOfTasks.GetTaskList()[_positionInTask].MarkAsDone();
+            }
+            else
+            {
+                _listOfTasks.GetTaskList()[_positionInTask].MarkAsUndone();
+            }
+        }
+
+        public static void ExecuteDeleteTask()
+        {
+            if (_positionInTask > 0)
+            {
+                _listOfTasks.DeleteTask(_positionInTask);
+                _positionInTask -= 1;
+            }
+            else
+            {
+                _listOfTasks.DeleteTask(_positionInTask);
+            }
+        }
+
+        public static void ExecuteChangePosition(int index, string direction)
+        {
+            if (direction == "up")
+            {
+                if (IsIndexValid(index, direction))
+                {
+                    _positionInTask -= 1;
+                }
+            }
+            else
+            {
+                if (IsIndexValid(index, direction))
+                {
+                    _positionInTask += 1;
+                }
+            }
+
+        }
+
+        public static string NavigationMenu()
+        {
+            var menuInfo = _menu.GetItems();
+
+            Console.ForegroundColor = ConsoleColor.Green;
+
+            return menuInfo.ToString();
+        }
+
+        public static void ShowTaskList()
+        {
+            List<Task> taskList = _listOfTasks.GetTaskList();
+
+            for (int i = 0; i < taskList.Count(); i++)
+            {
+                if (_positionInTask == i)
+                {
+                    Console.ForegroundColor = ConsoleColor.Blue;
+                }
+                else
+                {
+                    Console.ForegroundColor = ConsoleColor.Yellow;
+                }
+
+                var builder = new StringBuilder();
+                builder
+                    .Append($"{i + 1}")
+                    .AppendLine($"\tName: {taskList[i].Title}")
+                    .AppendLine($"\tDescription: {taskList[i].Description}")
+                    .AppendLine($"\tCreated At: {taskList[i].CreatedAt}")
+                    .AppendLine($"\tCompleted At: {(taskList[i].IsDone ? taskList[i].CompletedAt : "Not complite")}");
+
+                Console.WriteLine(builder.ToString());
+                if (!_listOfTasks.IsEmpty())
+                {
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    Console.WriteLine(new string('-', 40));
+                }
+            }
+        }
+
+        public static int GetNumberOfTask()
+        {
+            return _listOfTasks.GetTaskList().Count;

```

Comment:
Introduce a property "Count" on "taskList" type that returns the count of tasks from a List of tasks